core/templates/pages/story-editor-page/modal-templates/story-editor-unpublish-modal.component.html

```diff
@@ -1,15 +1,14 @@
 <div *ngIf="isSerialChapterFeatureFlagEnabled()">
   <div class="modal-header">
-    <span *ngIf="unpublishedChapters.length === 1">Unpublishing Chapter</span>
-    <span *ngIf="unpublishedChapters.length > 1">Unpublishing Chapters</span>
+    <span *ngIf="unpublishedChapters.length === 1">Unpublish Chapter</span>
+    <span *ngIf="unpublishedChapters.length > 1">Unpublish Chapters</span>
   </div>
   <div class="modal-body">
     <p>
       You're attempting to unpublish
-      <span *ngIf="unpublishedChapters.length === 1">chapter</span>
-      <span *ngIf="unpublishedChapters.length > 1">chapters</span>
-      <span *ngFor="let chapter of unpublishedChapters"> {{chapter}},</span>
-      please specify a reason and note the warning
+      <span *ngIf="unpublishedChapters.length === 1">chapter {{ unpublishedChapters[0] }}</span>
+      <span *ngIf="unpublishedChapters.length > 1">chapters {{ unpublishedChapters[0] }}-{{ unpublishedChapters[unpublishedChapters.length - 1] }}</span>
+      , please specify a reason and note the warning
```

Comment:
Again, please read this comment carefully: https://github.com/oppia/oppia/pull/18740#discussion_r1284414638 .

You're missing a period to break the run-on sentence up into two separate sentences.

Reply:
Done

Reply:
Done